hw/dv/tools/vcs/vcs.tcl

```diff
@@ -0,0 +1,44 @@
+# Copyright lowRISC contributors.
```

Comment:
`env::(TOOL)` will ensure that SHM is considered only when `TOOL == xcelium`. TCL files can be sourced with absolute paths. So you can source with `env::(TOOL_SRCS_DIR)` (which would also have to be set within the exports). See below comment.

Reply:
Done. Since we copy scripts into `TOOL_SRCS_DIR` (which is in the scratch directory), I also needed to add rules to copy the new `waves.tcl` as well as `$tool.tcl`, but I think this makes sense. I don't think I got a clear answer about where `waves.tcl` should live. It's currently at `hw/dv/tools/waves.tcl`, but if you think it should be in the data dir, I can easily switch it across.